docs/src/main/asciidoc/jreleaser.adoc

```diff
@@ -0,0 +1,838 @@
+create a Git release at the chosen repository, which includes tagging the repository, uploading the changelog, all
+distribution artifacts and their checksum as release assets.
+
+But before we do that let's add one additional feature, let's create a Homebrew formula that will make it easy for macOS
+users to consume the binary distribution, shall we?
+
+== Configuring Homebrew as a packager
+
+link:https://brew.sh/[Homebrew] is a popular choice among macOS users to install and manage binaries. Homebrew packages
+are at their core a Ruby file (known as a formula) that's executed on the target environment to install or upgrade a
+particular binary. JReleaser can create formulae from binary distributions such as the one we already have configured.
+
+For this to work we simply have to enable Homebrew in the JReleaser plugin configuration like so:
+
+[source,xml]
+----
+                <distributions>
+                  <app>
+                    <type>NATIVE_IMAGE</type>
+                    <brew>
+                      <active>ALWAYS</active>
+                    </brew>
+                    <artifacts>
+                      <artifact>
+                        <path>${distribution.directory}/{{distributionName}}-{{projectVersion}}-linux-x86_64.tar.gz</path>
+                        <platform>linux-x86_64</platform>
+                      </artifact>
+                      <artifact>
+                        <path>${distribution.directory}/{{distributionName}}-{{projectVersion}}-windows-x86_64.zip</path>
+                        <platform>windows-x86_64</platform>
+                      </artifact>
+                      <artifact>
+                        <path>${distribution.directory}/{{distributionName}}-{{projectVersion}}-osx-x86_64.zip</path>
+                        <platform>osx-x86_64</platform>
+                      </artifact>
+                    </artifacts>
+                  </app>
+                </distributions>
+----
+
+One last thing, it's a good practice to publish Homebrew formulae for non-snapshot releases thus change the project's version
+from `1.0.0-SNAPSHOT` to say `1.0.0.Alpha1` or go directly with `1.0.0` as you feel like doing. One last commit and we're done,
+say "feat: Add Homebrew packager configuration" as commit message. Alright, we're finally ready, let's post a release!
+
+== Creating a release
+
+It's been quite the whirlwind tour of adding configuration to the `pom.xml` but that's just for getting the project ready for
+its first release; subsequent release require less tampering with configuration. We can create a git release and the
+Homebrew formula with the `jreleaser:full-release` goal but if you still have some doubts on how things may play out then
+you can invoke the goal in dry-run mode that is, let JReleaser perform all local operations as needed without affecting
+remote resources such as Git repositories. This is how it would look like:
+
+[source,subs=attributes+]
```

Comment:
This listing was rendered very weirdly, I removed the `macro+` part and force pushed.